src/BenchmarkDotNet.Diagnostics.Windows/EtwProfiler.cs

```diff
@@ -138,6 +148,6 @@ private void Stop(DiagnoserActionParameters parameters)
         /// ETW sessions receive events with a slight delay.
         /// This increases the likelihood that all relevant events are processed by the collection thread by the time we are done with the benchmark.
         /// </summary>
-        private static void WaitForDelayedEvents() => Thread.Sleep(TimeSpan.FromMilliseconds(500));
+        private static void WaitForDelayedEvents() => Thread.Sleep(TimeSpan.FromMilliseconds(1000));
```

Comment:
@wojtpl2 is this change really needed? do we receive more events if we wait for 1s instead of 0.5s?

Reply:
No, I'm going to revert this change. It pushed it by my mistake.